trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py

```diff
@@ -138,21 +138,13 @@ def compute_inflow_and_tip_loss(r,R,Wa,Wt,B):
        F          tip loss factor                                                  [-]
        piece      output of a step in tip loss calculation (needed for residual)   [-]
     """
-    lamdaw            = r*Wa/(R*Wt)
-    lamdaw[lamdaw<0.] = 0.
-    f                 = (B/2.)*(1.-r/R)/lamdaw
-    f[f<0.]           = 0.
-    
-    piece             = np.exp(-f)
-    F                 = 2.*np.arccos(piece)/np.pi
-
-    Rtip = R
+    lamdaw             = r*Wa/(R*Wt)
+    lamdaw[lamdaw<=0.] = 1e-12
+
     et1, et2, et3, maxat = 1,1,1,-np.inf
```

Comment:
these are effectively not used

Reply:
Yep; they are tunable parameters that change the tip loss function. Can be adjusted, and may be updated in the future.

Reply:
maxat is not used. Default the rest as inputs above.

Reply:
done